gateway/gateway.go

```diff
@@ -788,11 +788,19 @@ func (gateway *HandleT) isUserSuppressed(workspaceID, userID, sourceID string) b
 	if !enableSuppressUserFeature || gateway.suppressUserHandler == nil {
 		return false
 	}
-	return gateway.suppressUserHandler.IsSuppressedUser(
+	isSuppressedUser := gateway.suppressUserHandler.IsSuppressedUser(
 		workspaceID,
 		userID,
 		sourceID,
 	)
+	if isSuppressedUser {
+		createdAt := gateway.suppressUserHandler.GetCreatedAt(workspaceID, userID, sourceID)
+		if !createdAt.IsZero() {
+			userSuppressionTimer := gateway.stats.NewTaggedStat("gateway.user_suppression_timer", stats.TimerType, stats.Tags{"workspaceID": workspaceID, "userID": userID, "sourceID": sourceID})
```

Comment:
Cardinality on userID would blow up the metrics

Reply:
Will remove it

Reply:
Another idea if we really need to capture details about userID could be that we can log if we get data from the suppressed userID after x number of day... say 60

@lvrach WDYT?

Reply:
@lvrach @atzoum